### ISSUE TITLE
Build Doris 0.11.25 thirtyparty on CentOS 7.5 Report Error and Exit

### ISSUE THREAD

**Issue by power225** [apache/incubator-doris#2507]
When build Doris 0.11.25 thirtyparty with GCC 5.3.1 on CentOS 7.5, it report error and exit:
-- Installing: /bak/incubator-doris-DORIS-0.11.25-release/thirdparty/installed/include/parquet/util/comparison.h
-- Installing: /bak/incubator-doris-DORIS-0.11.25-release/thirdparty/installed/include/parquet/util/crypto.h
-- Installing: /bak/incubator-doris-DORIS-0.11.25-release/thirdparty/installed/include/parquet/util/macros.h
-- Installing: /bak/incubator-doris-DORIS-0.11.25-release/thirdparty/installed/include/parquet/util/memory.h
-- Installing: /bak/incubator-doris-DORIS-0.11.25-release/thirdparty/installed/include/parquet/util/schema-util.h
-- Installing: /bak/incubator-doris-DORIS-0.11.25-release/thirdparty/installed/include/parquet/util/test-common.h
-- Installing: /bak/incubator-doris-DORIS-0.11.25-release/thirdparty/installed/include/parquet/util/visibility.h
-- Installing: /bak/incubator-doris-DORIS-0.11.25-release/thirdparty/installed/include/parquet/util/windows_compatibility.h
cp: cannot stat ‘./zstd_ep-install/lib/libzstd.a’: No such file or directory

How can I do with it?

**Comment by gilemagigi** [comment]
As I have tried in centos 7.6 with gcc-7.3.0, you can build **libzstd.a** with this method.
cd thirdparty/src/zstd-1.3.7/tests
make zstd-staticLib
mkdir -p apache-doris-0.12.0-incubating-src/thirdparty/src/arrow-apache-arrow-0.15.1/cpp/release/zstd_ep-install/lib64
cp ../lib/libzstd.a /PATH/TO/apache-doris-0.12.0-incubating-src/thirdparty/src/arrow-apache-arrow-0.15.1/cpp/release/zstd_ep-install/lib64

**Comment by EmmyMiao87** [comment]
Please build Doris by docker. http://doris.apache.org/master/zh-CN/installing/compilation.html#%E4%BD%BF%E7%94%A8-docker-%E5%BC%80%E5%8F%91%E9%95%9C%E5%83%8F%E7%BC%96%E8%AF%91%EF%BC%88%E6%8E%A8%E8%8D%90%EF%BC%89

**Comment by gilemagigi** [comment]
However, it's a problem that I use arm64/v8 to build doris, but I do not see the docker images with arm64. So do we have any plan to build docker images of arm64 arch?

**Comment by EmmyMiao87** [comment]
I understand that as long as you can run docker on the machine, it should be fine.

**Comment by gilemagigi** [comment]
Yeah, I just find the Dockerfile and will try to build a homebuild arm64 docker images to compile doris. Thanks
